source/fundamentals/monitoring.txt

```diff
@@ -0,0 +1,409 @@
+==========
+Monitoring
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+.. What is on this page?
+
+This guide explains how you can set up and configure **monitoring** in the
+MongoDB Java driver.
+
+.. What do any new terms mean?
+
+Monitoring is the process of getting information about the activities a running 
+program performs for use in an application or an application performance
```

Comment:
```suggestion
program performs to use in an application or an application performance
```

Reply:
I made a sentence that I think is equivalent to test this edit:

"Dish washing is the process of washing dishes for use in meals."
"Dish washing is the process of washing dishes to use in meals."

I looked around online and I think the main difference is that  "use in meals" becomes a noun with "for" and "use" becomes a verb with "to". 

In my head "for" seems like a slightly better fit, but I think both are correct. Because I can't see a reason to choose either one, I will go with the one I prefer and stick with "for".